src/app/code/community/Zendesk/Zendesk/Model/Customer.php

```diff
@@ -0,0 +1,39 @@
+<?php
+/**
+ * Created by PhpStorm.
+ * User: o5k4r1n
+ * Date: 25-05-18
+ * Time: 05:10 PM
+ */
+
+class Zendesk_Zendesk_Model_Customer extends Mage_Core_Model_Abstract{
+
+    public function syncronize(){
+        Mage::log('Cron Working', null, 'cron.log', true);
+        $customers = Mage::getModel('customer/customer')
+            ->getCollection()->setPageSize(90)->setCurPage(1);
```

Comment:
What would happen if a timeout is reached and not all customers have finished syncing? On the next cron run would the sync start from the beginning of the collection?

Reply:
If an timeout is reached  the syncronization will be made only with customers that doesn't have a zendesk_id (null), if the process is interrupted by any reason, then the syncronization will begin  then with the first customer found with no zendesk_id.  (zendesk_id = null)
It will not be necessary to start from zero this time.
If the process is interrupted will start again  in 10 minutes (for example)